<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userScreenClass</class>
 <widget class="QMainWindow" name="userScreenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="user_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>570</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="welcome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>491</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hi, User!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="display">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Display Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signout">
     <property name="geometry">
      <rect>
       <x>759</x>
       <y>460</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sign Out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteacc">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>300</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete Account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="display_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>570</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Full Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Age:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="display_back">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>440</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Governorate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>350</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vaccine Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>120</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>National ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>257</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gender:</string>
     </property>
    </widget>
    <widget class="QLabel" name="id">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>115</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>155</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="age">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>205</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="gender">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>252</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="gov">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>295</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="vaccine">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>345</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="delete_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>570</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="yes">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>290</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Are you sure you want to delete this account?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="no">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="edit_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>570</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="edit_vaccinated_1">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>310</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1 Dose</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">doseButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="male">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>197</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">genderButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>147</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Age:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_name">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="edit_vaccinated_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>330</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2 Doses</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">doseButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>287</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vaccine Status:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="female">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>197</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">genderButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>237</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Governorate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Full Name:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="edit_unvaccinated">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>290</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Unvaccinated</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">doseButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="edit_update">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>377</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_age">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>147</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_cancel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>377</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>194</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gender:</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_warn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>100</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>[Name is required!]</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="age_warn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>150</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>[Invalid age!]</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="edit_gov">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>240</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Alexandria</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aswan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Asyut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Beheira</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Beni Suef</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cairo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dakahlia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Damietta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Faiyum</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gharbia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Giza</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ismailia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kafr El Sheikh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Luxor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Matrouh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Minya</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Monufia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Valley</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>North Sinai</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Port Said</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Qalyubia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Qena</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red Sea</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sharqia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sohag</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>South Sinai</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Suez</string>
      </property>
     </item>
    </widget>
   </widget>
   <zorder>display_frame</zorder>
   <zorder>user_frame</zorder>
   <zorder>delete_frame</zorder>
   <zorder>edit_frame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="doseButtonGroup"/>
  <buttongroup name="genderButtonGroup"/>
 </buttongroups>
</ui>
